protocols/grpc/agent.proto

```diff
@@ -251,12 +251,13 @@ message CreateSandboxRequest {
 	string hostname = 1;
 	repeated string dns = 2;
 	repeated Storage storages = 3;
-	
+
```

Comment:
Nit: Whitespace damage.

Reply:
I deleted the extra whitespaces because it was unintentionally added by someone. I don't think we need white space there.

Reply:
ack